src/js/edu-benefits/5490/helpers.jsx

```diff
@@ -50,14 +50,8 @@ export const benefitsRelinquishedWarning = (
   </div>
 );
 
-export const benefitsDisclaimerSpouse = (
+export const benefitsDisclaimer = (
   <p>
-    IMPORTANT: If you are eligible for Chapter 35 Survivors’ and Dependents’ Educational Assistance Program (DEA) and eligible for Chapter 33 Post-9/11 GI Bill Marine Gunnery Sergeant John David Fry Scholarship (Fry Scholarship), you must relinquish entitlement to the benefit that you are not applying for (even if entitlement arises from separate events). <strong>You cannot retain eligibility for both programs simultaneously</strong>.
-  </p>
-);
-
-export const benefitsDisclaimerChild = (
```

Comment:
@peggygannon These were previously two different messages, is there now only one?

Reply:
@cehsu Hi, Claire, the message I am editing is the one at the top of the page: apply-for-education-benefits/application/5490/benefits/eligibility. IMPORTANT: If you are eligible for Chapter 35 Survivors’ and Dependents’ Educational Assistance Program (DEA) and eligible for Chapter 33 Post-9/11 GI Bill Marine Gunnery Sergeant John David Fry Scholarship (Fry Scholarship), you must relinquish entitlement to the benefit that you are not applying for (even if entitlement arises from separate events). You cannot retain eligibility for both programs simultaneously.

I only see one message there. I did not edit the conditional messages that pop up when you choose either DEA or Fry.

Reply:
@cehsu Hi, Claire, the message I'm editing is the one at the top of the page: 
IMPORTANT: If you are eligible for Chapter 35 Survivors’ and Dependents’ Educational Assistance Program (DEA) and eligible for Chapter 33 Post-9/11 GI Bill Marine Gunnery Sergeant John David Fry Scholarship (Fry Scholarship), you must relinquish entitlement to the benefit that you are not applying for (even if entitlement arises from separate events). You cannot retain eligibility for both programs simultaneously.

I didn't edit the conditional messages that pop up if the user chooses DEA or Fry.